backend/api/lib/index.ts

```diff
@@ -0,0 +1,15 @@
+import ServiceStack from "./serviceStack";
+import * as sst from "@serverless-stack/resources";
+
+export default function main(app: sst.App): void {
+  // Set default runtime and env variables for all functions
+  app.setDefaultFunctionProps({
+    runtime: "nodejs12.x",
+    environment: {
+      MAP_BOX_URL: process.env.MAP_BOX_URL ?? "",
```

Comment:
The function at `backend/api/src/location/getLocation.ts` will thrown an error, because it is being used to make a request to [mapbox](https://api.mapbox.com/geocoding/v5/mapbox.places) and get the coordinates based on the location.